pkg/kubectl/cmd/util/printing.go

```diff
@@ -81,7 +83,10 @@ func OutputVersion(cmd *cobra.Command, defaultVersion string) string {
 // Requires that printer flags have been added to cmd (see AddPrinterFlags).
 func PrinterForCommand(cmd *cobra.Command) (kubectl.ResourcePrinter, bool, error) {
 	outputFormat := GetFlagString(cmd, "output")
-	templateFile := GetFlagString(cmd, "template")
+	templateFile := GetFlagString(cmd, "golang-template")
```

Comment:
We eschew obscure abbreviations, but  I will counter the "it's too long" with:

-o go-tmpl="..."
-o jp-tmpl="..."

or 

-o template=jsonpath:"..."
-o template=go:"..."

not sure those are better.